src/containers/Product/AddProduct/index.js

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
-import { Button, Form, Input, InputNumber, Select, Switch } from 'antd';
+import {
+  Button,
+  Form,
+  Input,
+  InputNumber,
+  Select,
+  Switch,
+} from 'antd';
 import styles from './style.module.css';
 import classNames from 'classnames';
```

Comment:
Pula linha entre os seus imports e do react
```suggestion
import classNames from 'classnames';

import styles from './style.module.css';
```